simSPI/helpers.py

```diff
@@ -0,0 +1,218 @@
+"""Helper functions for tem.py."""
+import math
+import os
+
+import h5py
+import mdtraj as md
+import mrcfile
+import numpy as np
+from scipy.spatial.distance import pdist
+from scipy.stats import special_ortho_group
+
+
+def mrc2data(mrc_file=None):
```

Comment:
Why do we need this function to return None when nothing is given as input?

Reply:
good question, i'll remove this (I made it return None to pass the lint, but let's just remove this structure and assume the user passes in correct input).